packages/backend/src/AppModule.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Module } from '@nestjs/common';
+import { TodoModule, TodoScmReader } from '@backstage/plugin-todo-backend';
+import { getRootLogger } from '@backstage/backend-common/dist';
+
+// @ts-ignore this is actually not exported
+import { FetchUrlReader } from '@backstage/backend-common';
+import { Config } from '@backstage/config';
+
+@Module({})
+export class AppModule {
+  static register(config: Config) {
+    return {
+      imports: [
+        TodoModule.register({
+          scmReaderImpl: TodoScmReader.fromConfig(config, {
+            logger: getRootLogger(),
+            reader: new FetchUrlReader(),
```

Comment:
I'd be a bit interested to see how some of this wiring would work together with the shared utilities from `backend-common`. Although perhaps that ends up pretty boring, just additional inject and imported constants?